src/dbnode/persist/schema/types.go

```diff
@@ -29,6 +29,11 @@ import (
 // tooling needs to upgrade older files to newer files before a server restart
 const MajorVersion = 1
 
+// MinorVersion is the minor schema version for a set of fileset files.
+// This is only incremented when *non-breaking* changes are introduced that
+// we want to have some level of control around how they're rolled out.
+const MinorVersion = 1
```

Comment:
nit: Do we want to perhaps start at minor version 0? Can't remember if we actually started at 0 for major version or not (should check history).

Reply:
It looks like MajorVersion started at 1 (commit 4afa833), but I think 0 makes sense here since I'm just going to end up bumping it to 1 once I'm ready to roll out the new checksumming behavior.